packages/pagination/src/elements/Pagination.spec.tsx

```diff
@@ -8,10 +8,16 @@
 import React from 'react';
 import { render, fireEvent } from 'garden-test-utils';
 import { KEY_CODES } from '@zendeskgarden/container-utilities';
-
-import Pagination, { IPaginationProps, PAGE_TYPE } from './Pagination';
+import { default as Pagination, IPaginationProps, PAGE_TYPE } from './Pagination';
```

Comment:
Should we use the standard default export? i.e.

```ts
import Pagination, { IPaginationProps, PAGE_TYPE } from '...';
```

Reply:
No, without `default as ...` we miss the ability to test the `ref` exported via the default `withTheme` export.